mindmaps-graql/src/main/java/io/mindmaps/graql/internal/reasoner/query/Query.java

```diff
@@ -134,6 +136,24 @@ protected Query(Atomic atom) {
     @Override
     public Conjunction<PatternAdmin> getPattern(){ return pattern;}
 
+    public boolean isRuleResolvable(){
+        boolean ruleResolvable = false;
+        Iterator<Atomic> it = atomSet.iterator();
+        while(it.hasNext() && !ruleResolvable)
+            ruleResolvable = it.next().isRuleResolvable();
+
+        return ruleResolvable;
+    }
+
+    public QueryAnswers getAnswers(){ throw new IllegalStateException(ErrorMessage.ANSWER_ERROR.getMessage());}
```

Comment:
What's the point of these methods if all they do it throw exceptions ? Maybe there is a better way to structure things so we don't need to do this ?